src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  let objectStyles = sourceString.split(';');
+
+  objectStyles = objectStyles
+    .map(word => word.split(':'))
+    .reduce((obj, item) => {
+      if (item.length === 2) {
```

Comment:
maybe, you can do this solution an another way (without if)

Reply:
agree, it would be better to avoid such hardcoding